releasenotes/notes/0.15/quadratic-form-circuit-f0ddb1694960cbc1.yaml

```diff
@@ -0,0 +1,22 @@
+---
+features:
+  - |
+    A new class, :class:`~qiskit.circuit.library.QuadraticForm` to the
+    :mod:`qiskit.circuit.library` module for implementing a a quadratic form on
+    binary variables. The circuit library element implements the operation
+
+    .. math::
+
+      |x\rangle |0\rangle \mapsto |x\rangle |Q(x) \mod 2^m\rangle
+
+    for the quadratic form :math:`Q` and :math:`m` output qubits.
+    The quadratic form is specified using a matrix for the quadratic
+    terms, a vector for the linear terms and a constant offset. For example::
```

Comment:
```suggestion
    The quadratic form is specified using a matrix for the quadratic
    terms, a vector for the linear terms and a constant offset.
    If all terms are integers, the circuit implements the quadratic form
    exactly, otherwise it is only an approximation.
    
    For example:
```